etna/packages/etna-js/hooks/useReduxState.js

```diff
@@ -0,0 +1,44 @@
+import {useContext, useState, useEffect} from 'react';
+import {ReactReduxContext} from "react-redux";
+
+const ident = (state) => state;
+
+export function useReduxState(selectorFunction = ident) {
```

Comment:
Again, React 'hooks' alternative to the `connect` function.  Greatly reduces boiler plat, instead of having to do things like

```
function MyComponent({ a, b, c, d, e, f }) {}
connect(MyComponent, (state) => { a: 1, b: 2, c: 3, d: 4, e: 5 })
```

Another benefit of using hooks here is that `connect` only allows components to withdraw state from the redux, which means the component then has to pass those values around all over the place.  Hooks, on the other hand, can use other hooks, including this one, meaning you can peel back seperate parts of the system that engage with the react state and don't have to require the host component itself pass around a ton of parameters and configure itself correctly with `connect`.